r2-streamer-swift/Parser/EPUB/EPUBParser.swift

```diff
@@ -111,10 +111,26 @@ final public class EpubParser: PublicationParser {
         if ((try? container.data(relativePath: EPUBConstant.lcplFilePath)) != nil) {
             return DRM(brand: .lcp)
         }
-        
-        return DRM(brand: .adobe)
-        
-        // return nil
+        return nil
+    }
+  
+    
+    /// Check license files at the book location.
+    /// Adobe or LCP DRM licence can be a separate file, not included into .epub archive.
+    /// - Parameter url: File URL
+    /// - Returns: A DRM, if any found.
+    private static func scanForDRM(at url: URL) -> DRM? {
+        // LCP DRM license file has .lcpl extension
+        let lcpPath = url.path.replacingOccurrences(of: ".\(url.pathExtension)", with: ".lcpl")
+        if FileManager.default.fileExists(atPath: lcpPath) {
+            return DRM(brand: .lcp)
+        }
+        // Adobe DRM license file has "_rights.xml" suffix, as defined in ADEPT library
+        let adobePath = "\(url.path)_rights.xml"
```

Comment:
can you define this hardcoded string suffix somewhere so that it's consistent everywhere? Maybe it makes sense to define it here on in r2-shared? you probably know the best place.

Reply:
the best place imo is DRM.Brand in R2Shared. moved it there.